stats.cpp

```diff
@@ -1,5 +1,25 @@
 #include "stats.h"
+#include <limits>
 
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
+Stats Statistics::ComputeStatistics(const std::vector<double>& statsVec) {
```

Comment:
Stats is inside a namespace, so it's giving a compilation error here. You will either need to prefix it with the namespace here, or do a `using`